node/variables.tf

```diff
@@ -218,3 +244,15 @@ variable "ff_wait_ready" {
   type    = bool
   default = false
 }
+
+variable "ff_with_kubeproxy" {
+  type = bool
+}
+
+variable "server_node_labels" {
```

Comment:
why not using the same name for server and agent labels? what about annotations - we need those also